bodo/pandas/_plan.h

```diff
@@ -303,6 +309,26 @@ duckdb::unique_ptr<duckdb::LogicalFilter> make_filter(
     std::unique_ptr<duckdb::LogicalOperator> &source,
     std::unique_ptr<duckdb::Expression> &filter_expr);
 
+/**
+ * @brief Create a limit node.
+ *
+ * @param source - the source of the data to be filtered
+ * @param n - the number of rows to return
+ * @return duckdb::unique_ptr<duckdb::LogicalLimit> - the limit node
+ */
+duckdb::unique_ptr<duckdb::LogicalLimit> make_limit(
+    std::unique_ptr<duckdb::LogicalOperator> &source, int n);
+
+/**
+ * @brief Create a limit node.
+ *
+ * @param source - the source of the data to be filtered
+ * @param n - the number of rows to return
```

Comment:
Isn't `n` a percentage value?

Reply:
When I did this I was thinking of using sample to do head since it supports pushdown to the get nodes and that is why it is an int and an absolute number but given our change in approach we could change this to a real percentage.  Duckdb supports both modes (and a couple more I think that I don't remember at the moment) so you have a choice.